app/app.go

```diff
@@ -202,6 +208,12 @@ type RegenApp struct {
 	sm *module.SimulationManager
 }
 
+// WasmWrapper allows us to use namespacing in the config file
+// This is only used for parsing in the app, x/wasm expects WasmConfig
+type WasmWrapper struct {
+	Wasm wasm.Config `mapstructure:"wasm"`
+}
```

Comment:
Do we need it? If yes then please move it to a new file.

Reply:
Yes, we need this for `wasmConfig` which is needed for `wasm.NewKeeper` func arguement.

Reply:
Yes, we need this for `wasmConfig` which is needed for `wasm.NewKeeper` func arguement.